client/executor/src/integration_tests/mod.rs

```diff
@@ -47,6 +47,26 @@ fn call_in_wasm<E: Externalities>(
 		ext,
 		code,
 		heap_pages,
+		false,
```

Comment:
Won't the other tests fails because of missing imports? :D

Reply:
done (but I think wasmtime is failing at instantiation because of the missing function. I will fix that)